src/ui/YouTube/YouTubeThumbnail.ts

```diff
@@ -28,22 +33,28 @@ export class YouTubeThumbnail extends Component {
    * @componentOptions
    */
   static options: IYouTubeThumbnailOptions = {
+
     /**
-     * Specify the width that the thumbnail should have.
+     * Specifies the width (in pixels) of the YouTube thumbnail.
      *
-     * The default value is `200px`
+     * Default value is `200px`.
      */
```

Comment:
In the generated doc site, the markup configuration example is:
`data-height='foo'`
If possible, it would be better to display something like
`data-height='100px`